release/update-rsync.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
+#
+# Update rsync server directories with latest bundled source code
+# This will update rsync to the latest snapshot OR release tarball
+# available in the www service
+#
+
+test -e ~/.server.config && . ~/.server.config || exit 1
+
+cd `dirname $0`/..
+
+for version in `ls -1 $SQUID_VCS_PATH | grep squid | cut -d- -f2`; do
+	rm -rf tmp
+	mkdir -p tmp/squid-$version
```

Comment:
To clarify and reduce noise, I suggest moving this command _after_ the loop-and-skip code below, to the `tar` command that actually needs this directory.

Reply:
Bug fixes welcome in followup PRs. This one is scoped at importing the current status-quo automation.